uber/templates/registration/form.html

```diff
@@ -270,7 +284,6 @@
         <label for="shirt" class="col-sm-2 control-label">Shirt Size</label>
         <div class="col-sm-6">
             <select name="shirt" class="form-control">
```

Comment:
We don't need this option on the admin form, since ``c.SHIRT_OPTS`` already includes the ``0`` option.